home/tests/test_models.py

```diff
@@ -730,7 +730,7 @@ def test_simple_template_submission(self) -> None:
         wat.save()
         wat.save_revision()
 
-        wat.submit_whatsapp_template(None)
+        # submit_whatsapp_template(wat)
```

Comment:
We can remove these comments, similar to how we removed the function call in the other tests.